src/ui/StackedBarChart.js

```diff
@@ -938,9 +970,9 @@ WGL.ui.StackedBarChart = function(m, div_id, x_label, filterId, params, category
     svg.selectAll('.y.axis').transition().duration(30)
       .call(yAxis);
 
-    svg.selectAll(".selected.bar").attr("d", barPathSelected);
-    svg.selectAll(".unselected.bar").attr("d", barPathUnselected);
-    svg.selectAll(".out.bar").attr("d", barPathOut);
+    svg.selectAll(".selected.bar").attr("d", barPathSelected).attr("fill", "#ffa91b");
+    svg.selectAll(".unselected.bar").attr("d", barPathUnselected).attr("fill", "#8CC5F9");
+    svg.selectAll(".out.bar").attr("d", barPathOut).attr("fill", "#e3e4e4");
   }
```

Comment:
I guess we might reuse these colors so what about to better change the value of barPathSelected then hardcoding this.